core/server/master/src/main/java/alluxio/master/file/options/CreatePathOptions.java

```diff
@@ -43,6 +49,7 @@ protected CreatePathOptions() {
     mOwner = "";
     mGroup = "";
     mMode = Mode.defaults();
+    mAcl = Lists.newArrayList();
```

Comment:
use this? https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyList()

Reply:
done